build/ProjectReunion-CI.yml

```diff
@@ -0,0 +1,85 @@
+# see https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases for info on yaml ADO jobs
+name: $(BuildDefinitionName)_$(date:yyMM).$(date:dd)$(rev:rrr)
+variables:
+  minimumExpectedTestsExecutedCount: 3000  # Sanity check for minimum expected tests to be reported
```

Comment:
Same here, 3000 might be overkill right now.

Reply:
thanks, done.